### ISSUE TITLE
visitorKeys does not contain `assertions` property.

### ISSUE THREAD

**Issue by ota-meshi** [typescript-eslint/typescript-eslint#4177]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/TROUBLESHOOTING.md) and my problem is not listed.

**Repro**

```TS
import foo from "mod" assert { type: "json" };
```

**Expected Result**

I expected that the implementation of the plugin rule could traverse the ImportAttribute node.

**Actual Result**

It is not actually traversed.

**Additional Info**

**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `5.4.0` |
| `TypeScript`                | `4.5.0` |
| `ESLint`                    | `8.0.0` |
| `node`                      | `14` |

**Comment by bradzacher** [comment]
oops - this was missed in #4074
